src/Cpp/1-getting-started/1-HelloWindow/Application.hpp

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include <cstdint>
+#include <memory>
+#include <string_view>
+
+// ReSharper disable once CppInconsistentNaming
```

Comment:
Comment seems out of place?

Reply:
Nope thats because of `GLFWwindow` Resharper wants it to be named `GlfwWindow`  same for the other CppInconsistentMappin occurences

Reply:
Hmm is it something that should exist in this codebase though? It sounds like a you-problem rather than something that should be viewed by people without specific plugins installed (for example, I do not have Resharper)

Reply:
It is a problem for all people who dont have ReSharper, Some might have it. I dont see its a problem for those who dont, since its a comment, it also provides enough context what it is about, imho.

However, we can probably somehow possibly have it lint that kind of stuff with clang-lint? (I will keep ReSharper hints in for the time being)